scripts/tests/test_validate_docstrings.py

```diff
@@ -564,6 +591,11 @@ def test_bad_generic_functions(self, func):
         assert errors
 
     @pytest.mark.parametrize("klass,func,msgs", [
+        # See Also tests
+        ('BadSeeAlso', 'desc_no_period',
+         ('Missing period at end of description',)),
+        ('BadSeeAlso', 'desc_first_letter_lowercase',
+         ('should be capitalized',)),
```

Comment:
Can you test that the element reported is the element with the problem? Meaning that `Series.iloc` is ok, and in the error message we want to have that this is for `Series.tail`.